### PR TITLE
Regex issues on some methods.

### PR DESCRIPTION
**Pull request by ParvinYagubzada**
1. In getDefaultString() and getDefaultStrNum() Regex had small issue. Regex in getDefaultNumber() replaced with new Regex.
Overall project was readable, good designed and well thought out. Congratulations!

### CODE REVIEW

File: Homework/src/main/java/az/code/util/Util.java [hunk 1]
```diff
@@ -107,7 +107,7 @@ public static String getDefaultNumber(String info) {
         System.out.println("Enter " + info + ":");
         while (true) {
             String str = new Scanner(System.in).next();
-            if (str.matches("([+]*[0-9]+|!back)")) {
+            if (str.matches("^\\+994(50|51|55|70|77|99)[1-9]\\d{6}$")) {
```
Comment (ElgunMaqsudzade) [inline Homework/src/main/java/az/code/util/Util.java:110]: Hello Parvin. I have a string (!back) that i am using for returning to the menu every time. You should add it here. Thanks)

File: Homework/src/main/java/az/code/util/Util.java [hunk 2]
```diff
@@ -180,7 +180,7 @@ public static boolean getYesNo(String qna) {
         System.out.println("Enter yes/no:");
         while (true) {
             String str = new Scanner(System.in).next();
-            if (str.matches("([ynYN]|(yes|no|Yes|No|YES|NO))")) {
+            if (str.matches("(?i)([yn]|(?i)(yes|no))")) {
```
Comment (ElgunMaqsudzade) [inline Homework/src/main/java/az/code/util/Util.java:183]: Quite a good work here. I didn't know about ignoreCase syntax. Thank you very much

**Review by ElgunMaqsudzade — COMMENTED** [review]
Please check the review out bro)

**Comment by ParvinYagubzada** [comment]
Thanks for feedback. I changed regex after your request.

**Review by ElgunMaqsudzade — COMMENTED** [review]
Quite a Good Job :)

**Pull request merged**